ImprovedAnvil/lib/creatures/02_patches_vanilla.tph

```diff
@@ -86,10 +86,26 @@ COPY_EXISTING ~dragblac.cre~ ~override/dragblac.cre~
   ADD_CRE_ITEM ~S!trap~ #0 #0 #0 ~UNSTEALABLE&UNDROPPABLE~ ~BELT~
   ADD_CRE_ITEM ~S!dblack~ #0 #0 #0 ~UNSTEALABLE&UNDROPPABLE~ ~GLOVES~
   ADD_CRE_ITEM ~minhp1~ #0 #0 #0 ~UNSTEALABLE&UNDROPPABLE~ ~AMULET~
+  ADD_CRE_ITEM ~s!birchim~ #0 #0 #0 ~UNSTEALABLE&UNDROPPABLE~ ~BOOTS~  
   ADD_CRE_ITEM ~S!misc02~ #0 #0 #0 ~UNSTEALABLE~ ~INV8~
   ADD_CRE_ITEM ~S!scrl06~ #0 #0 #0 ~UNSTEALABLE~ ~INV9~
   ADD_CRE_ITEM ~S!scrl27~ #0 #0 #0 ~UNSTEALABLE~ ~INV10~
 
+COPY_EXISTING ~fsdragon.cre~ ~override/fsdragon.cre~ //Ancient Dragon
+  ADD_CRE_ITEM ~s!birchim~ #0 #0 #0 ~UNSTEALABLE&UNDROPPABLE~ ~BOOTS~  
+  
+COPY_EXISTING ~dragblue.cre~ ~override/dragblue.cre~ //Abazigal
+  ADD_CRE_ITEM ~s!birchim~ #0 #0 #0 ~UNSTEALABLE&UNDROPPABLE~ ~BOOTS~  
+
+COPY_EXISTING ~bazdra02.cre~ ~override/bazdra02.cre~ //Draconis
```

Comment:
The 02_patches_vanilla.tph is only for vanilla creatures. Draco Ferrous and other IA-added dragons should be in the appropriate part of the code. Please, respect the structure of the mod :-)

Also, don't bother with Draco Ferrous, he is from the unfinished first version of the druid stronghold and most probably won't make it into our newly designed quest.

Reply:
sorry, i'm really not trying to annoy you or screw up the codebase... you may have missed the comment in the pull request but its not clear to me where the IA dragon patches should go

Reply:
No worries, I am not annoyed, just obsessive after doing this for many years professionally :-)